docs/community_snippets.md

```diff
@@ -312,6 +313,35 @@ Use the `#region` for organizing your code (including good code folding).
 }
 ```
 
+### Send-MailMessage
+
+Add the Send-MailMessage cmdlet with the most common parameters in a hashtable for splatting, by @fullenw1.
+
+#### Snippet
+
+```json
+"Send-MailMessage": { 
```

Comment:
I would recommend naming the snippet something other than the command name.  This can get confusing when you think you're autocompleting the command name but then get this snippet.  We have been using a naming convention in the extension where "examples" start with `ex-`.  Perhaps you call this snippet `ex-send-mail-message`?